test.md

```diff
@@ -1,16 +1,11 @@
-
 # Test File
 
 This file is for testing per-line commenting.
 
-You can do per-line comments that are associated with a commit, or with a pull request.
-
-To comment on an older line, copy a link to that line and then paste the link into a commit comment.
-
-Or use blame to find the commit where the line was added, and add the comment to that commit.
+* You can do per-line comments that are associated with a commit, or with a pull request.
+* To comment on an older line, copy a link to that line and then paste the link into a commit comment.
```

Comment:
I don’t know if you were aware of this, but when GitHub is displaying code with line numbers, you can click on the line number and the URL becomes a direct link to that line, like this:

https://github.com/dondi/bazaar/blob/master/boxes-with-physics/boxes-with-physics.js#L21

(nothing in particular there, just a random file I picked out of one of my public repos)

You can also shift-click across a range of lines for a multi-line link.